### ISSUE TITLE
Questions about time in automations and temp sensors

### ISSUE THREAD

**Issue by kabturek** [bieniu/home-assistant-config#1]
Hey, gj with your config - the script to update current temp on those z-wave thermostat is great!
I'm using some of your tricks/ideas right now to spice up my config.
I have some questions:
1. In heating automatations you use 
`(now().strftime("%s") | int | timestamp_custom("%H:%M")) `
is there a reason why you don't use?
`now().strftime("%H:%M") `

2. in your xiaomi temperature sensors why do you substract half a degree from the reading ? 
`(states.sensor.temperature_158d0002273538.state | float - 0.5) | round(1)`

Cheers

**Comment by bieniu** [comment]
@kabturek Hi, thanks for kind words.
Regarding your questions:
1. Honestly, I do not remember why I used such complicated syntax. I wrote it over a year ago and it looks bad today. `now().strftime("%H:%M")` works of course. I will change it soon ;-)
2. I compared temperature measurements from Xiaomi sensors with a thermometer, which is quite accurate and usually the temperature on the xiaomi sensors was 0.3-0.5 degrees higher. I don't like the dual sensor solution and I'm looking to improve it without changing component's code.
